raiden/ui/cli.py

```diff
@@ -483,15 +488,18 @@ def version(short):
     help="Which Ethereum client to run for the smoketests",
 )
 @click.pass_context
-def smoketest(ctx, debug, eth_client):
+def smoketest(ctx, debug, eth_client, report_path):
```

Comment:
Would be nice to add type annotations while touching this.

Reply:
done